docs/_data/nav.yml

```diff
@@ -6,9 +6,9 @@
   href: /support/
   category: support
 
-- title: Blog
-  href: /blog/
-  category: blog
+# - title: Blog
+#   href: /blog/
+#   category: blog
```

Comment:
Actually, not sure `/` will work, maybe you could use an absolute URL `https://nuclide.io`. You can experiment :)